usmqe_tests/api/gluster/test_volume.py

```diff
@@ -63,10 +66,9 @@ def test_create_volume_invalid(
         valid_cluster_id,
         invalid_volume_configuration)["job_id"]
     # TODO check correctly server response or etcd job status
-    api.wait_for_job_status(
-            job_id,
-            status="failed",
-            issue="https://github.com/Tendrl/tendrl-api/issues/33")
+    api.wait_for_job_status(job_id,
+                            status="failed",
+                            issue="https://github.com/Tendrl/tendrl-api/issues/33")
```

Comment:
Remove? Really?
That doesn't sound right. There are two possibilities:
1. the parameter is not really needed and it should be part of appropriate `pytest.check` or part of code in the `wait_for_job_status` method. The parameter should never exist.
2. the parameter is needed as we are expecting more issues could happen